content/en/tracing/setup/go.md

```diff
@@ -139,6 +139,35 @@ func main() {
 
 For more tracer settings, see available options in the [configuration documentation][57].
 
+### B3 Headers Extraction and Injection
+
+Datadog APM tracer supports [B3 headers extraction][58] and injection for distributed tracing.
+
+Distributed headers injection and extraction is controlled by
+configuring injection/extraction styles. Currently two styles are
+supported:
+
+* Datadog: `Datadog`
+* B3: `B3`
+
+Injection styles can be configured using the environment Variable
+`DD_PROPAGATION_STYLE_INJECTION=Datadog,B3`
```

Comment:
It's called 
```suggestion
`DD_PROPAGATION_STYLE_INJECT=Datadog,B3`
```

Reply:
Huh, sorry about that. Not sure how I missed this. I `grep`'d to find this to begin with.